x-pack/plugins/observability/public/pages/alert_details/components/page_title.tsx

```diff
@@ -70,7 +72,13 @@ export function PageTitle({ alert, dataTestSubj }: PageTitleProps) {
       <EuiFlexGroup direction="row" alignItems="center" gutterSize="xl">
         <EuiFlexItem grow={false}>
           <AlertLifecycleStatusBadge
-            alertStatus={alert.active ? ALERT_STATUS_ACTIVE : ALERT_STATUS_RECOVERED}
+            alertStatus={
+              alert.fields[ALERT_STATUS] === 'active'
```

Comment:
@benakansara I have seen Flapping status in the UI as well. Do we need to consider this status as well?

Reply:
@mgiota The flapping status is already covered by `AlertLifecycleStatusBadge` component.

<img width="1267" alt="Screenshot 2023-12-18 at 19 31 40" src="https://github.com/elastic/kibana/assets/69037875/ed9a029c-001c-4b7e-a920-3c3836ab2b26">